apps/avatax/src/modules/avatax/order-confirmed/avatax-order-confirmed-payload-transformer.ts

```diff
@@ -51,8 +51,9 @@ export class AvataxOrderConfirmedPayloadTransformer {
         commit: avataxConfig.isAutocommit,
         addresses: {
           shipFrom: avataxAddressFactory.fromChannelAddress(avataxConfig.address),
-          // billing or shipping address?
-          shipTo: avataxAddressFactory.fromSaleorAddress(order.billingAddress!),
+          shipTo: avataxAddressFactory.fromSaleorAddress(
+            order.shippingAddress ?? order.billingAddress!,
```

Comment:
Please dont use `!`

Reply:
One of these should always be present - I added payload verification but Typescript is not cooperating with me today so for now `!` will have to stay.